pages/linux/nsnake.md

```diff
@@ -0,0 +1,24 @@
+# nsnake
+
+> Snake game in terminal.
```

Comment:
Perhaps this will read better?
```suggestion
> Snake game in the terminal.
```

Reply:
@sbrl I recently made pull request for `snake4` #4895 .

Most of the arguments were similar, so I took that as a boilerplate.